code/modules/vending/medical.dm

```diff
@@ -11,7 +11,7 @@
 					/obj/item/healthanalyzer = 4,
 					/obj/item/reagent_containers/pill/patch/styptic = 5,
 					/obj/item/reagent_containers/pill/patch/silver_sulf = 5,
-					/obj/item/reagent_containers/pill/salbutamol = 2,
+					/obj/item/reagent_containers/syringe/perfluorodecalin = 2,
```

Comment:
why not keep some salbut in the vendor? maybe premium?

Reply:
no

Reply:
no one is gonna ever make salbut, it has a pain in the ass recipe that isnt even efficiently macroable and does one of the least usefull heals add a premium pill so people wont forget it exists

Reply:
it's still in space tobacco so it doesn't need to be in here. If it's so useless then perf should work fine without you ever needing to get this in the first place.

I have no intention of adding it to premium. I'd prefer premium vendor items to be unique items rather than "pay to not put in work"